elements/lisk-p2p/test/integration/p2p.ts

```diff
@@ -1847,4 +1847,130 @@ describe('Integration tests for P2P library', () => {
 			it.skip('should not evict peers with high responseRate', async () => {});
 		});
 	});
+
+	describe('Connected network: Minimum peer discovery threshold is set to 1', () => {
```

Comment:
Please, try to merge both scenarios into one single describe. Perhaps even add them to one of the existing discovery describes.

Reply:
Both scenarios require new configurations which could alter behavior of nodes. Therefore it wouldn't fit into other previous describes. I can put both describe blocks in a wrapper describe.